_data/monthly-calls.yml

```
@@ -1,10 +1,22 @@
 videos:
+  - title: A first look at U.S. Web Design System Web Components
+    subtitle: See new Web Components in action
+    description: |
+      USWDS Product Lead Dan Williams talked about the alpha stage for Web Components and design principles supporting the work. Engineers James Mejia and Matt Henry from USWDS did a live demo with examples of working alpha Web Components and shared a behind-the-scenes look at how they’re built.
+    date: June 2024
+    id: 
```

@finekatie It looks like the [June Monthly Call](https://www.youtube.com/watch?v=yBCLyWO0yCo&list=PLd9b-GuOJ3nGqDYCNsCMHCQ9MdD5jfB01&index=1) is live on YouTube. Can we add that here?
```suggestion
    id: yBCLyWO0yCo?si=NBOx_cexXn8j-CDk
```

---

hi @amyleadem , thank you for spotting that! They have asked us _not_ to add monthly call links until the video is up on their event page. It may not have been fully edited yet or have transcript finished. I'm not sure why it ends up on YouTube before they're finished with it, but that's the rule of thumb for now.  I will check with them to see if they're about ready to add it, but this is a hold for now.

---

I have checked in. They're still waiting for offical transcript to be completed, so we should wait and add this later. I had to do this with the May monthly call and that's why the link is being added in this PR. How do I reject the requested change?

---

Update: June video was added to the events page. I added :)